src/app.ts

```diff
@@ -83,6 +83,18 @@ export function generateServerlessRouter(
 
     mainRouter.use(setServerUrlMiddleware(fhirConfig));
 
+    // Set default content-type to 'application/fhir+json'
+    mainRouter.use(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
Can you move this middleware to it's own file under https://github.com/awslabs/fhir-works-on-aws-routing/tree/mainline/src/router/middlewares?
It feels a bit better organized that way.